client/widgets/MiscWidgets.cpp

```diff
@@ -261,7 +261,7 @@ void CArmyTooltip::init(const InfoAboutArmy &army)
 {
 	OBJECT_CONSTRUCTION;
 
-	title = std::make_shared<CLabel>(66, 2, FONT_SMALL, ETextAlignment::TOPLEFT, Colors::WHITE, army.name);
+	title = std::make_shared<CLabel>(66, 3, FONT_SMALL, ETextAlignment::TOPLEFT, Colors::WHITE, army.name);
```

Comment:
Might be better to switch to CENTERLEFT alignment or to BOTTOMLEFT and adjust coordinates.

Your solution would fix fonts in your particular version of game with your particular fonts (English gog?), but might not work in other versions, including other languages from gog

Reply:
How exactly works these alignments? I was thinking now with center set and max width it will be rendered from coordinates to both sides. It looks like it works like that from 7 vs 13 for example, both seems to be centered.

Also mentioned change from 2 to 3, I tried too, but it then overlap from bottom. These two windows looks same, but coordinates are moved from some reason. And probably by 1 exactly. No idea why.

I have tried English, Czech and Russian translation mods. Can check also others.